.github/CODEOWNERS

```diff
@@ -161,6 +161,8 @@
 /modules/programs/sbt.nix                             @kubukoz
 /tests/modules/programs/sbt                           @kubukoz
 
+/modules/programs/senpai.nix                          @malte-v
+
```

Comment:
"se" should be after "sc", so this should go after `/modules/programs/scmpuff.nix`

Reply:
Fixed